### PR TITLE
Update feature-request.md

### PR DESCRIPTION
**Pull request by roaksoax**
Update feature-request.md to provide a nicer description.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature Request
-about: Request a new feature we haven't thought of
+about: Request a new feature or an enhancement of a current one
```
Comment (karenzone) [inline .github/ISSUE_TEMPLATE/feature-request.md:3]:
```suggestion
about: Request a new feature or suggest an enhancement to a existing one
```

**Review by karenzone — CHANGES_REQUESTED** [review]
For consideration

**Review by karenzone — APPROVED** [review]
LGTM 🎉

File: .github/ISSUE_TEMPLATE/feature-request.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Feature Request
-about: Request a new feature we haven't thought of
+about: Request a new feature or suggest an enhancement to a existing one
```
Comment (hendry-lim) [inline .github/ISSUE_TEMPLATE/feature-request.md:3]:
```suggestion
about: Request a new feature or suggest an enhancement to an existing one
```

**Review by hendry-lim — CHANGES_REQUESTED** [review]

**Pull request merged**